src/openrct2/peep/Peep.h

```diff
@@ -52,12 +52,12 @@ struct Ride;
 class GameActionResult;
 using ParkEntranceIndex = uint8_t;
 
-enum PeepType : uint8_t
+enum class PeepType : uint8_t
 {
-    PEEP_TYPE_GUEST,
-    PEEP_TYPE_STAFF,
+    Guest,
+    Staff,
 
-    PEEP_TYPE_INVALID = 0xFF
+    Invalid = 0xFF
 };
```

Comment:
I'd have hoped we'd add a namespace here:

```c++
namespace Peep
{
    enum class Type : uint8_t
    {
        Guest,
        Staff,
        Invalid = 0xFF
    };
}
```

Can we do it for the following? Seems like a very good opportunity to increase our usage of namespaces.

Reply:
I'm trying to figure out how to turn this into a namespace but Peep is clashing with the struct Peep lower in the file and giving me an error.  Any idea how to refactor around that?